.travis.yml

```diff
@@ -5,3 +5,5 @@ node_js:
   - "0.12"
   - "0.10"
 sudo: false
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
I'm not a huge fan either. Here is what they point to for a code review https://github.com/codecov/codecov-bash. I'm actually not sure what kind of harm this could do outside of messing with Travis' servers. It's not going to run in our local build pipeline.